components/Banner/src/MDCBannerView.m

```diff
@@ -327,13 +327,14 @@ - (void)deactivateAllConstraints {
 - (CGSize)sizeThatFits:(CGSize)size {
   MDCBannerViewLayoutStyle layoutStyle = [self layoutStyleForSizeToFit:size];
   CGFloat frameHeight = 0.0f;
+  CGFloat widthLimit = size.width;
```

Comment:
Tip: This code looks like it's duplicated in `layoutStyleForSizeToFit:`. Can you refactor a bit to eliminate the duplication and provide a reusable method?

```objc
- (CGSize)contentSizeForLayoutSize:(CGSize)layoutSize {
  CGFloat remainingWidth = layoutSize.width;
  CGFloat marginsPadding = self.layoutMargins.left + self.layoutMargins.right;
  remainingWidth -= marginsPadding;
  remainingWidth -= (kLeadingPadding + kTrailingPadding);
  return CGSizeMake(remainingWidth, layoutSize.height);
}
```

Reply:
Good suggestion. Addressed in 0f55b0f4ad62e9feb898b5334df7eb7aeea65da9.